src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -488,16 +543,48 @@ public override void Dispose()
             Dispose(false);
         }
 
+        private async Task FlushAcceptQueue()
+        {
+            try {
+                // Writer may or may not be completed.
+                _state.AcceptQueue.Writer.Complete();
+            } catch { };
```

Comment:
NIT:
```suggestion
            try 
            {
                // Writer may or may not be completed.
                _state.AcceptQueue.Writer.Complete();
            }
            catch
            { }
```

Reply:
Just use `TryComplete` instead of `try/catch`.